SECURITY.md

```diff
@@ -0,0 +1,5 @@
+# Reporting a Vulnerability
+
+Please, [open a draft security advisory](https://github.com/spring-projects/security-advisories/security/advisories/new) if you need to disclose and discuss a security issue in private with the Spring Boot team. Note that we only accept reports against [supported versions](https://spring.io/projects/spring-boot#support).
+
+For more details, check out our [security policy](https://spring.io/security-policy).
```

Comment:
I don't think we need this as we already inherit the policy from spring-projects/.github. We can just link to https://github.com/spring-projects/spring-boot/security/policy.

Reply:
Added to `contact_links` but the issue page changed a bit.
<img width="1182" alt="Screenshot 2024-10-17 at 19 07 42" src="https://github.com/user-attachments/assets/f7fbf077-67ff-4a55-af79-0d9014782555">

Reply:
Report a security vulnerability option should be inherited from https://github.com/spring-projects/.github automatically.